swayfmt/tests/mod.rs

```diff
@@ -79,6 +79,69 @@ pub struct Foo<T, P> {
         &mut formatter,
     );
 }
+
+#[test]
+fn struct_alignment_with_public_fields() {
+    let mut formatter = Formatter::default();
+    formatter.config.structures.field_alignment = FieldAlignment::AlignFields(40);
+
+    check_with_formatter(
+        r#"contract;
+pub struct Foo<T, P> {
+   barbazfoo: u64,
+   pub baz     : bool,
+}
+"#,
+        r#"contract;
+pub struct Foo<T, P> {
+    barbazfoo : u64,
```

Comment:
I feel like it should be formatted like this as it would be in rust.

```suggestion
pub struct Foo<T, P> {
    barbazfoo: u64,
    pub baz: bool
}
```

Reply:
In the case of `FieldAlignment::AlignFields` there is a space between the field name and the `:`. This is clear from the implementation, in _item_struct/mod.rs_, line 116:
```
// Add `:`, ty & `CommaToken`
write!(
    formatted_code,
    " {} ",
    type_field.colon_token.span().as_str(),
)?;
```
This formatting is also expected in the `struct_alignment` test. Adding the `pub` keyword just followed the existing expectations in case of  `FieldAlignment::AlignFields`.